consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/validation/RoundChangeMessageValidator.java

```diff
@@ -94,6 +96,13 @@ public boolean validate(final RoundChange msg) {
   }
 
   private boolean validateBlock(final Block block) {
+
+    final BlockValidator blockValidator =
```

Comment:
Can also use ProtocolSchedule.getByBlockHeader here as we have the block header

Reply:
Done 👍